src/Pages/JobseekerData.js

```diff
@@ -0,0 +1,180 @@
+import React, { useState } from 'react';
+import './NavigatorDashboard.css';
+import Checkboxes from './Checkboxes';
+import { updateJobseeker } from '../Services/jobseeker-service';
+
+// later we can make each tab a different component, the individual tabs take a jobseeker as a prob
+
+function Onboard() {
+  // eslint-disable-next-line no-unused-vars
+
+  const [jobseeker, setJobseeker] = useState({
+    name: 'Solia Tennis', // from login
+    pronouns: 'Pronouns',
+    phone: 'Phone',
+    email: 'solia@bestpl.us', // from login
+    ethnicity: 'Ethnicity',
+    degree: 'No',
+    degreeType: 'None',
+    certificate: 'No',
+    certificateType: 'None',
+  });
+
+  const skills = ['Accounting Software', 'Administrative', 'Adobe Software Suite', 'Bilingual', 'Brand Management', 'Cold Calling']; // can expand to all skills on spreadsheet
```

Comment:
I think alan's filter component has the complete skills checklist that can be added!